### DISCUSSION TITLE
US citizen traveling abroad stuck on trying to reuse sensor

### DISCUSSION THREAD

**Discussion by gitanO0** [NightscoutFoundation/xDrip#3150] [Q&A] [answered]
I'm using G6 sensor and latest android nightly drip+ release. I'm traveling outside the US and am trying to reuse a sensor that just expired. But no matter what I do I get a "failed to start sensor" error. Now the app seems to be in some sort of limbo state, not started and not stopped, just displaying that error. I do have a new sensor that I can use but am worried now about how to get back up and running either on the old sensor or on the new one.

**Comment by Navid200** [comment]
Can you post a screenshot of the G5/G6/G7 status page?  
https://navid200.github.io/xDrip/docs/StatusG5G6.html

If you want, you can blackout the transmitter ID.  But, please don't blackout the firmware ID.

**Reply by gitanO0** [reply]
Due to the late hour here and need to get some rest, I decided in haste to put the new sensor on and try to start a brand new session. So far it is working, stating that I have 1.5 hours left of warmup. Not sure if this will succeed or it will fail but so far so good.  If it works I will delete this discussion. If not, I'll proceed as instructed by @Navid200 to post the status screen.

**Reply by Navid200** [reply]
There is no reason to delete a discussion.

Instructions for restart are here: https://navid200.github.io/xDrip/docs/Restart-G6-sensor.html

**Reply by gitanO0** [reply]
I did all that correctly and have done this "reuse" process many times but never after starting a sensor in the US then stopped and restarted abroad. The status screen looked all normal except for the "sensor status" parameter stated "failed to start sensor".

In any event, I'm now at 1.1 hours left of warmup on a new sensor so not feeling too bad about it.

**Comment by Navid200** [comment, marked as answer]
If you need to start a sensor, but, xDrip only has the option to stop, you can do the following.
Wait for a read cycle to complete.  Then, on the status page, you will see last connected: 20 seconds.  This means you have more than 4 minutes before the next read.  Now, tap on stop sensor.  Right away, follow this to clear the queue:
https://navid200.github.io/xDrip/docs/Clear-queue.html

After you do that, xDrip will have a start option.  But, the transmitter never receives the stop command.

If you have a sensor that has stopped and you want to restart, you will need to remove the transmitter following the restart instructions and it is very important to keep the transmitter out for at least 10 minutes.  If you don't do that, after you start, the transmitter will trigger a restart attempt detection and fail.
The requirement to keep the transmitter out for at least  10 minutes is the case even with a brand new sensor, and it is not a requirement imposed by xDrip.  It is imposed by the transmitter.

I have travelled a lot with a G6 and have never experienced any problems caused by time zone change.
A G6 transmitter has no knowledge of your location.  It has no knowledge of time zones.  All it has is a local oscillator that it uses as its clock.  
The only time you could experience problem with going to a different location is when using Dexcom app with G7, not G6.
But, even with G7, xDrip does not impose a local restriction.

It could haven just been a coincidence that you experienced a problem at the same time you were travelling.

**Reply by gitanO0** [reply]
Thanks. That all makes sense. I might not have waited the full 10+ min with the transmitter out.
